index.html

```diff
@@ -213,58 +216,70 @@ <h1>Lorem ipsum dolor sit amet consectetur adipisicing elit<span
         <div class="footer">
             <h1 class="footer-text">iPaidThat</h1>
             <div class="footer-content">
-                <div class="row">
+                <div class="row ">
```

Comment:
Is this space intentional?

Reply:
No, must've been a mis-click on the space